gabc/index.html

```diff
@@ -146,7 +148,69 @@ <h3 id="header">Header</h3>
 <code>score-copyright: (C) Abbaye de Solesmes, 1934</code>
 </dt>
 
+<dd><a name="author">author</a></dd>
+<dt>The author of the piece, if known; of course, the author of most
+traditional chant is not known.</dt>
+
+<dd><a name="language">language</a></dd>
+<dt>The language of the lyrics.</dt>
+
+<dd><a name="language">oriscus-orientation</a></dd>
+<dt>If <code>legacy</code>, the orientation of an unconnected oriscus must be set manually.</dt>
+
+<dd><a name="mode">mode</a></dd>
+<dt>The mode of the piece. This should normally be an arabic
+	number between 1 and 8, but may be any text required for unusual
+	cases. The mode number will be converted to roman numerals and
+	placed above the initial unless one of the following conditions are
+	met:
+	<ul>
+	<li>There is a <code>\greannotation=</code> defined immediately prior to <code>\gregorioscore</code>.</li>
+	<li>The <code>annotation</code> header field is defined.
+	</ul>
+</dt>
+
+<dd><a name="mode-modifier">mode-modifier</a></dd>
+<dt>The mode "modifier" of the piece. This may be any
+	<span class="tex">T<span class="epsilon">e</span>X</span> code to typeset after the mode, if the mode is typeset.  If the mode
+	is not typeset, the mode-modifier will also not be typeset.
+</dt>
+
+<dd><a = name="mode-differentia">mode-differentia</a></dd>
+<dt>The mode or tone differentia of the piece.  Typically,
+	this expresses the variant of the psalm tone to use for the piece.  This may
+	be any <span class="tex">T<span class="epsilon">e</span>X</span> code to typeset after the mode-modifier, if the mode is typeset.
+	If the mode is not typeset, the mode-differentia will also not be typeset.
+</dt>
+
+<dd><a name="annotation">annotation</a></dd>
+<dt>The annotation is the text to appear above the initial letter.
+Usually this is an abbreviation of the office-part in the upper line,
+and an indication of the mode (and differentia for antiphons) in the
+lower.  Either one or two annotation fields may be used; if two are
+used, the first is the upper line, the second the lower.  Example:<br />
+<blockquote><code>annotation:Ad Magnif.;<br />
+annotation:VIII G;</code></blockquote>
+
+Full <span class="tex">T<span class="epsilon">e</span>X</span> markup is accepted:
+
+<blockquote><code>annotation:{\color{red}Ad Magnif.};<br />
+annotation:{\color{red}VIII G};</code></blockquote>
+
+If the user already defined annotation(s) in the main <span class="tex">T<span class="epsilon">e</span>X</span> file via
+	<code>\greannotation</code> then the <code>annotation</code> header field will not
+	overwrite that definition.
+</dt>
 
+<dd><a name="staff-lines">staff-lines</a></dd>
+<dt>The number of lines on the staff.  You can have as few as 2 lines and as many as 5.  If this header is left out, then the default will be the classic 4 line staff.
+</dt>
+
+</dl>
+
+<p>Although gregorio ascribes no special meaning to them, other suggested headers are:</p>
```

Comment:
I ran into this when doing the TeX documentation.  I think "gregorio" should be capitalized here as it refers to the system on a whole as opposed to the "gregorio" executable.  I've been trying to be consistent in "Gregorio" meaning the system as a whole, "GregorioTeX" being the tex portion, and "gregorio" being the executable.  What do you think?

Reply:
I think that's a reasonable standard and will fix this.